x/supplier/types/codec.go

```diff
@@ -31,6 +31,8 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
 
 	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
+
+	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
```

Comment:
Not a blocker but just curious if we know what's going on here. It looks redundant at first glance here.

Reply:
Updated.

This is a result of the `pocket`/`github.com/pokt-network/potroll` translation.

1) The first one is intended.
2) The second one was a result of not deleting it after the Claim scaffolding.
3) The third one is a result of the Proof scaffolding

I'm deleting (2) & (3).